.github/workflows/aws-cloud-deploy.yml

```diff
@@ -176,20 +201,228 @@ jobs:
+
+      - name: ECR
+        if: ${{ inputs.deploy-container && env.cloud_aws_access_secret != '' }}
+        run: |
+          echo "Project ECR - ${{ vars.PROJECT_AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.region }}.amazonaws.com" 
+          echo "Brightspot Cloud ECR - ${{ vars.CLOUD_AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.region }}.amazonaws.com"
+
+      - name: Login to project ECR
+        uses: docker/login-action@v3
+        if: ${{ inputs.deploy-container && env.cloud_aws_access_secret != '' }}
+        with:
+            registry: ${{ vars.PROJECT_AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.region }}.amazonaws.com
+
+      - name: Login to Brightspot Cloud ECR
+        uses: docker/login-action@v3
+        if: ${{ inputs.deploy-container && env.cloud_aws_access_secret != '' }}
+        with:
+            registry: ${{ vars.CLOUD_AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.region }}.amazonaws.com
+
+
+      - name: Build Container
+        id: build-container
+        if: ${{ inputs.deploy-container && env.cloud_aws_access_secret != '' }}
         shell: bash
+        run: |
+          version=""
+          build=""
+
+          echo "GITHUB_ACTIONS_TAG ${GITHUB_ACTIONS_TAG}"
+          echo "GITHUB_ACTIONS_PULL_REQUEST ${GITHUB_ACTIONS_PULL_REQUEST}"
+
+          if [[ ! -z "$GITHUB_ACTIONS_PULL_REQUEST" && "$GITHUB_ACTIONS_PULL_REQUEST" != "" ]]; then
+              version="pullrequest-$GITHUB_ACTIONS_PULL_REQUEST"
+              build=pullrequest-$GITHUB_ACTIONS_PULL_REQUEST-build$GITHUB_RUN_NUMBER
+          elif [[ "$GITHUB_ACTIONS_TAG" =~ ^v[0-9]+\. ]]; then
+              echo "GITHUB_ACTIONS_TAG ${GITHUB_ACTIONS_TAG}"
+              version=release-${GITHUB_ACTIONS_TAG/v/}
+              build=release-${GITHUB_ACTIONS_TAG/v/}
+          else
+              COMMIT_COUNT=$(git rev-list --count HEAD)
+              COMMIT_SHA=$(git rev-parse --short=6 HEAD)
+              build=development-$(git describe --all | sed 's/heads\///' | sed 's/\//-/g')
+              build+=-build$GITHUB_RUN_NUMBER
+              version=development-$(git describe --all | sed 's/heads\///' | sed 's/\//-/g')
+          fi
+
+          cp brightspot-build/*.war etc/container/web.war
+          cd etc/container
+
+          PROJECT="${{ inputs.project }}"
+          REPOSITORY="${{ inputs.repository }}"
+
+          if [ -z ${REPOSITORY} ]; then
+             REPOSITORY="$PROJECT/$PROJECT"
+          fi
+
+          REGISTRY="${{ vars.PROJECT_AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.region }}.amazonaws.com"
+          BUILD_TAG="$REGISTRY/$REPOSITORY:$build-ARM64"
+          VERSION_TAG="$REGISTRY/$REPOSITORY:$version-ARM64"
+
+          docker buildx build --platform=linux/arm64 --push --build-arg WAR_ARTIFACT=web.war -t $VERSION_TAG -t $BUILD_TAG -f Dockerfile --metadata-file docker-metadata.json .
+
+          echo "container-build-tag=$(echo $BUILD_TAG)" >> $GITHUB_OUTPUT
+          echo "container-version-tag=$(echo $VERSION_TAG)" >> $GITHUB_OUTPUT
+
```

Comment:
x86 build has another step to publish Brightspot version - is that not needed for arm since it only needs to be done once? what if x86 is skipped?

Reply:
correct, though good point. Currently we don't let you skip either build, but we should add that in the near future and move the version calculation to the manifest area. I'm going to try the later now.

Reply:
good idea!